src/main/java/graphql/nadel/Nadel.java

```diff
@@ -132,9 +132,7 @@ private DefinitionRegistry createCommonTypes() {
     }
 
     private GraphQLSchema createOverallSchema() {
-        List<DefinitionRegistry> registries = this.services.stream()
-                .map(Service::getDefinitionRegistry)
-                .collect(toList());
+        List<DefinitionRegistry> registries = FpKit.map(this.services, Service::getDefinitionRegistry);
```

Comment:
@andimarek - in the most Immutable branch I have removed FpKit.map and moved it to ImmutablesKit

Should I put it back?  Of fix it here in Nadel later?

Reply:
@bbakerman Nadel is still on a quite "old" graphql-java version. See https://github.com/graphql-java/nadel/pull/164
We should merge this and then use ImmutablesKit.

Reply:
K